torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -947,11 +986,15 @@ void AliasDb::analyzeSubgraph(Node* node) {
   // subgraph block.
   TORCH_INTERNAL_ASSERT(
       subgraphBlock->outputs().size() >= node->outputs().size());
-  for (const auto i : c10::irange(node->outputs().size())) {
+  for (size_t i = 0; i < node->outputs().size(); i++) {
```

Comment:
nit: any reason not to use irange?

Reply:
there are too many C++ idioms to remember 😢